openassessment/xblock/static/sass/oa/views/_oa-base.scss

```diff
@@ -1086,7 +1086,7 @@
     }
 
     .action--save {
-      @extend %action-button
+      @extend %action-button !optional
     }
 
     .feedback {
```

Comment:
Is there CSS that should be deleted related to AI grading? I'm thinking of the Studio editors, as well as the LMS view.